pep-0002.txt

```diff
@@ -2,196 +2,62 @@ PEP: 2
 Title: Procedure for Adding New Modules
 Version: $Revision$
 Last-Modified: $Date$
-Author: Martijn Faassen <faassen@infrae.com>
-Status: Superseded
+Author: Brett Cannon <brett@python.org>,
+        Martijn Faassen <faassen@infrae.com>
+Status: Active
 Type: Process
 Content-Type: text/x-rst
 Created: 07-Jul-2001
 Post-History: 07-Jul-2001, 09-Mar-2002
 
 
-
-PEP Replacement
-===============
-
-This PEP has been superseded by the updated material in the Python
-Developer's Guide [1]_.
```

Comment:
If this PEP was superseded, leave it as a historical document, and create a new one?

Reply:
Indeed, very much agreed, see [my comment](https://github.com/python/peps/pull/2651#pullrequestreview-1019030218) to that effect.